test/known_problems/should_pass/inner_union_subtype_of_root_union.erl

```diff
@@ -0,0 +1,18 @@
+-module(inner_union_subtype_of_root_union).
+
+-export([f/0]).
+
+% The problem is that {t, a} | {t, b} is not subtype of {t, a|b}.
```

Comment:
I'm wrong here. It's the other way around. `{t, a|b}` is (in our implementation) not subtype of `{t, a} | {t, b}`. I'll fix this.

Reply:
Fixed.